pkgs/applications/misc/plover/default.nix

```diff
@@ -1,37 +1,138 @@
-{ lib, fetchFromGitHub, python3Packages, wmctrl, qtbase, mkDerivationWith }:
+{
+  lib,
+  fetchFromGitHub,
+  python310Packages,
+  qt5,
+  mkDerivationWith,
+}:
 
 {
-  stable = throw "plover.stable was removed because it used Python 2. Use plover.dev instead."; # added 2022-06-05
+  dev =
+    let
+      inherit (python310Packages)
+        appdirs
+        babel
+        buildPythonPackage
+        mock
+        pyqt5
+        pyserial
+        pytestCheckHook
+        setuptools
+        wcwidth
+        xlib
+        ;
+      plover_stroke = mkDerivationWith buildPythonPackage rec {
+        pname = "plover_stroke";
+        version = "1.1.0";
 
-  dev = with python3Packages; mkDerivationWith buildPythonPackage rec {
-    pname = "plover";
-    version = "4.0.0.dev10";
+        pyproject = true;
 
-    meta = with lib; {
-      broken = stdenv.hostPlatform.isDarwin;
-      description = "OpenSteno Plover stenography software";
-      maintainers = with maintainers; [ twey kovirobi ];
-      license     = licenses.gpl2;
-    };
+        src = fetchFromGitHub {
+          owner = "openstenoproject";
+          repo = "plover_stroke";
+          rev = version;
+          sha256 = "sha256-A75OMzmEn0VmDAvmQCp6/7uptxzwWJTwsih3kWlYioA=";
+        };
 
-    src = fetchFromGitHub {
-      owner = "openstenoproject";
-      repo = "plover";
-      rev = "v${version}";
-      sha256 = "sha256-oJ7+R3ZWhUbNTTAw1AfMg2ur8vW1XEbsa5FgSTam1Ns=";
-    };
+        propagatedBuildInputs = [ setuptools ];
+
+        nativeCheckInputs = [ pytestCheckHook ];
+        pythonImportsCheck = [ "plover_stroke" ];
+
+        meta =
+          let
+            inherit (lib) licenses platforms maintainers;
+          in
```

Comment:
Honestly, I would just do `lib.licenses`, `lib.platforms`, etc. Four characters is not exactly that much

Reply:
Since I use the same inherit for each `meta`, I've moved it to the top. So there is only a single `inherit (lib) ...`